scripts/ci/nightly-lines.py

```
@@ -53,19 +57,23 @@ def head_sha(repo: str, line: str, gh) -> str:
+    if isinstance(data, list):
+        data = data[0] if data else {}
+    content = base64.b64decode(data.get("content", "") or "").decode("utf-8", errors="replace")
+    return PROFILE_INPUT in content
 
 
 def dispatch_large(line: str) -> None:
     # `gh` печатает адрес прогона в stdout, а stdout этого скрипта — строки
     # для GITHUB_OUTPUT: чужая строка там — отказ раннера. Адрес — в stderr.
     completed = subprocess.run(
-        ["gh", "workflow", "run", LARGE_WORKFLOW, "--ref", line], check=True, capture_output=True, text=True
+        ["gh", "workflow", "run", LARGE_WORKFLOW, "--ref", line, "-f", "profile=large"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)